frontend/__tests__/services/loadChallengesById.spec.js

```diff
@@ -0,0 +1,24 @@
+import MockAdapter from 'axios-mock-adapter';
+import loadChallengesById from '../../src/services/loadChallengesById';
+import api from '../../src/services/api';
+
+const apiMock = new MockAdapter(api, { delayResponse: 1000 });
+
+describe(loadChallengesById, () => {
+    const mock = {
+        id: 'any-id',
+        response: {
+            data: [{ id: 'right-id' }, { id: 'wrong-id' }],
+        },
+    };
+
+    beforeEach(() => {
+        apiMock.onGet(`/challenges/${mock.id}`).reply(200, mock.response.data);
+    });
+
+    it('should get the data from api with right id', async () => {
+        // eslint-disable-next-line camelcase
```

Comment:
Since camelcase shows itself to be a frequent warning, I suggest we disable it or name our prop in camelcase.

@stevescruz @Lorenalgm What do you think?